components/availability/src/Availability.svelte

```diff
@@ -1684,87 +1218,21 @@
                     startTime: new Date(slot.start_time).toLocaleString(),
                   }}
                   class="slot {slot.selectionStatus} {slot.availability}"
-                  class:pending={slot.selectionPending}
-                  class:hovering={slot.hovering}
+                  class:pending={event_to_hover && slot.selectionPending}
                   data-start-time={new Date(slot.start_time).toLocaleString()}
                   data-end-time={new Date(slot.end_time).toLocaleString()}
-                  on:mousedown={(event) => {
-                    if (!touchPriority)
-                      handleSlotInteractionStart({ event, slot, day });
-                  }}
-                  on:mouseenter={(event) => {
-                    if (!touchPriority) handleSlotHover({ event, slot, day });
-                  }}
-                  on:mouseleave={() => (slot.hovering = false)}
-                  on:mouseup={() => {
-                    if (!touchPriority && mouseIsDown)
-                      handleSlotInteractionEnd(day);
-                  }}
-                  on:keypress={(e) => {
-                    if (e.code === "Space" || e.code === "Enter") {
-                      startDrag(slot, day);
-                      tick().then(() => endDrag(day));
-                    }
```

Comment:
How does this new interaction change accommodate accessibility if keyboard support is being removed?

Reply:
It wasn't working at all for consecutive booking, which is what the default selection now mimics. This is something we'll likely be adding back, but it would need to be reworked entirely to support the new selection mode. Likely post-MVP release.